packages/migrations/src/actions/2023.07.24T10.10.04.schema_sdl_checksums.ts

```diff
@@ -0,0 +1,31 @@
+import { type MigrationExecutor } from '../pg-migrator';
+
+export default {
+  name: '2023.07.24T10.10.04.schema_sdl_checksums.sql',
+  run: ({ sql }) => sql`
+CREATE TABLE IF NOT EXISTS "public"."schema_sdl_checksums" (
+    "checksum" text NOT NULL,
+    "schema_sdl" text NOT NULL,
+    "target_id" uuid NOT NULL REFERENCES "targets" ("id") ON DELETE CASCADE,
+    "created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    "updated_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    PRIMARY KEY(target_id, checksum));
```

Comment:
if we go for the `"schema_sdl_checksums"` path, we also need to make sure that we delete `"schema_sdl_checksums"` automatically, once they are no longer referenced by either a `schema_versions` or `schema_checks` table entry.